### ISSUE TITLE
Settings out of focus?

### ISSUE THREAD

**Issue by suhan-paradkar** [cloneconf/nedium#6]
Looks like the settings are still under development.....
But great job on the app...
![Screenshot_2022-01-15-08-57-30-10](https://user-images.githubusercontent.com/76830230/149607259-10fb7302-dcf0-43d8-b6d5-b5d1ab004112.jpg)

**Comment by suhan-paradkar** [comment]
Well.. I just downloaded the new release for nedium...
I observe that that the font size os too big... any way to adjust font size?

![Screenshot_2022-01-17-13-30-55-44](https://user-images.githubusercontent.com/76830230/149730514-34a2887e-a45a-486e-a5f0-70c291f07f9e.jpg)

**Comment by suhan-paradkar** [comment]
> If you have some settings you find it important to add, please let me know ;)

Well... about that... I guess it would be best to concentrate on adding customization options for reader mode... Since the app is primarily used for reading articles...

**Comment by cloneconf** [comment]
Yeah the font overall is messed up currently, hopefully I will fix it ;)
